core/modules/system/system.module

```diff
@@ -964,6 +978,17 @@ function system_menu() {
     'file' => 'system.admin.inc',
     'weight' => -20,
   );
+  if (settings_get('backup_directory') !== FALSE) {
+    $items['admin/config/system/backup-restore'] = array(
+      'title' => 'Site backups',
+      'description' => 'Change site backup settings and view current backups.',
+      'page callback' => 'backdrop_get_form',
+      'page arguments' => array('system_backup_settings'),
```

Comment:
Hm. On admin/config/system/backup-restore I get a hard error:

```

    Warning: Undefined array key "system_backup_settings" in backdrop_retrieve_form() (line 822 of /var/www/bdroptesting/html/core/includes/form.inc).
    TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, function "system_backup_settings" not found or invalid function name in backdrop_retrieve_form() (line 866 of /var/www/bdroptesting/html/core/includes/form.inc).
```

WIP, I guess. as there's really no such function, yet. :stuck_out_tongue_winking_eye: I better wait, until this form's implemented.

Reply:
Yeah, I started on one but now I think it's unnecessary. I'll remove this from hook_menu().